pkg/apis/config/default.go

```diff
@@ -151,6 +156,10 @@ func NewDefaultsFromMap(cfgMap map[string]string) (*Defaults, error) {
 		}
 		tc.DefaultMaxMatrixCombinationsCount = int(matrixCombinationsCount)
 	}
+	if defaultForbiddenEnvString, ok := cfgMap[defaultForbiddenEnvs]; ok {
+		fEnvs := strings.Split(defaultForbiddenEnvString, ", ")
+		tc.DefaultForbiddenEnvs = fEnvs
+	}
```

Comment:
NIT: can we please split on just comma instead of comma along with a space to allow variables without following such strict guidelines?

Also, can we replace an array of strings with `sets.NewString()` followed by `Insert()` to filter accidental duplicates.

Reply:
splitting with ", ", includes splitting with comma/comma+space. By splitting only with comma, there may be an extra space that needs to removed from each item.